frontend/src/components/CurrentBooking.test.tsx

```diff
@@ -1,7 +1,62 @@
+// @ts-nocheck
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import CurrentBooking from './CurrentBooking';
+import * as bookingService from '../services/bookingService';
+import { getBookings } from '../services/bookingService';
+import userEvent from '@testing-library/user-event';
+import ReactDOM, { unmountComponentAtNode } from 'react-dom';
 
-test('renders CurrentBooking.tsx', () => {
-    expect(true).toBe(true);
+const fakeBooking = [
+    {
+        id: '123',
+        startTime: '2021-10-21T17:32:28Z',
+        endTime: '2021-10-21T19:32:28Z',
+        room: {
+            id: 'c_188fib500s84uis7kcpb6dfm93v25@resource.calendar.google.com',
+            name: 'Amor',
+            capacity: 4,
+            features: ['Jabra', 'TV', 'Webcam'],
+            nextCalendarEvent: '2021-10-21T17:32:28Z'
+        }
+    }
+];
+
+let container = null;
```

Comment:
Tästä alaspäin vois wrapata describellä, sil voi ryhmitellä testejä
```
describe('CurrentBooking', () => {
   beforeEach
   ...
})
```

Ehkä tää on vähä makuasia, ei oo pakko muokata.